test/shared/integrationDiffTest.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+#EOC
+#------------------------------------------------------------------------------
+#                  GEOS-Chem Global Chemical Transport Model                  !
+#------------------------------------------------------------------------------
+#BOP
+#
+# !MODULE: integrationDiffTest.sh
+#
+# !DESCRIPTION: Looks for differences in two integration test or parallel
```

Comment:
If it can be applied to either integration or parallel tests could we rename it to diffTest.sh to be more generic?

Reply:
Right now it only can difference integration tests.  I can work on this if you wish.